libsolidity/analysis/TypeChecker.cpp

```diff
@@ -322,6 +322,14 @@ bool TypeChecker::visit(StructDefinition const& _struct)
 	return false;
 }
 
+bool TypeChecker::visit(ModifierDefinition const& _modifier)
+{
+	if (!_modifier.isImplemented() && !_modifier.virtualSemantics())
```

Comment:
Need to think if _modifier.body().accept(*this) is needed, just like _function.body().accept(*this)